script.js

```diff
@@ -0,0 +1,50 @@
+let startBtn = document.querySelector("#startButton");
+startBtn.addEventListener("click", playGame);
+//ash says we can use this as a global object
+let gameState = ["", "", "", "", "", "", "", "", ""];
+
+function decideTurn(currentPlayer) {
+    if (currentPlayer === 'o') {
+        return 'x';
+    }   else  {
+        return  'o';
+    }
+}
+function addToScreen(tile, currentPlayer) {
+    tile.innerHTML = currentPlayer;
+}
+function alterGameState(gameState, clicked, currentPlayer) {
+    if(gameState[clicked] == "") {
+        gameState[clicked] = currentPlayer;
+        return true;
+    } else {
+        return false;
+    }
+}
```

Comment:
No need for the else condition when returning in the if.

Take a look at this example - it achieves the exact same thing but is a bit simpler
```js
function alterGameState(gameState, clicked, currentPlayer) {
    if(gameState[clicked] == "") {
        gameState[clicked] = currentPlayer;
        return true;
    } 
    return false;
}
```